lab_3_generate_by_ngrams/main.py

```diff
@@ -105,13 +104,9 @@ def get_token(self, element_id: int) -> Optional[str]:
         """
         if not isinstance(element_id, int):
             return None
+        if element_id not in self._storage.values()):
+            return None
```

Comment:
where is the imlementation?

Reply:
You removed it. Right now only checks  are on the function

Reply:
it's here now